### PR TITLE
Minor UI Testing

### PR DESCRIPTION
**Pull request by karosas**
I screwed up my fork a bit, resending clean pull request.
Original message:
Hi there, decided to contribute a little and noticed you don't have any tests (Shame) so I wrote a few UI tests using Espresso.

Added couple of dependencies for android test app.
Added couple app's build.gradle entries to configure test app with espresso and enable coverage output
Coverage report is in /app/build/reports/coverage/debug
If you don't need any of this, just ignore it :)

P.S. I'm super new to contributing to OS, so if I made some mistakes related to git then please let me know.

Screwed up pull request https://github.com/vilnius/tvarkau-vilniu/pull/68

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -149,6 +156,13 @@ dependencies {
     compile('com.crashlytics.sdk.android:crashlytics:2.6.2@aar') {
         transitive = true;
     }
+
+    testCompile "org.mockito:mockito-core:$mockito"
+    androidTestCompile "com.android.support.test.espresso:espresso-core:$espresso"
+    androidTestCompile "com.android.support.test.espresso:espresso-intents:$espresso"
+    androidTestCompile "com.android.support.test:rules:$testRunner"
+    androidTestCompile "com.android.support.test:runner:$testRunner"
+    androidTestCompile "com.android.support:support-v4:24.1.1" // 23.4.0 causes dependency error
```
Comment (vycius) [inline app/build.gradle:165]: Created PR to fix this build error https://github.com/vilnius/tvarkau-vilniu/pull/69

**Comment by vycius** [comment]
Update `.travis.yml` to perform UI testing :)

**Comment by inetaa** [comment]
@karosas Hi, thank you for joining us! Really nice initiative with integration tests. :) I have one comment regarding your commit names - please use common naming conventions. Great article on it's benefits and usage can be found here: http://chris.beams.io/posts/git-commit/). 
- Use the imperative tone in the subject line - describe what a commit does, rather than what it did, so instead of "added something" should be "add something", in addition capitalize the subject line "Add something". 
- If possible be more specific what the code does rather than being general. E.g. "Add basic UI tests" could be "Add basic UI tests for Main, Profile and About activitites". This should still fit in character limit. 
- For describing what commit does, normally verbs are used ("Fix, Upgrade, Add, Implement, ect"), so don't leave commits without description what it does as in "versionCode mistake". Better option would be "Fix versionCode mistake" but there can be many mistakes and it does not tell what the code does, so an even better name would be "Downgrade versionCode to 6" - in this case you actually know what the code does without looking into it. And if you need more explanation, you can add in the commit body. 

That's it. Hope to see more commits of yours!

**Comment by ViliusKraujutis** [comment]
@karosas, "This branch is out-of-date with the base branch" could you please rebase it to latest master? Then I guess it would be good to go to master. Also, please take a look to @vycius comment regarding #69

**Comment by ViliusKraujutis** [comment]
PR is deprecated, so closing it. //cc @karosas

**Pull request closed without merging**